DialogflowApp.ts

```diff
@@ -36,35 +42,40 @@ export class DialogflowApp extends App implements IPostMessageSent, IPostLivecha
         super(info, logger, accessors);
     }
 
-    public async onEnable(environment: IEnvironmentRead, configurationModify: IConfigurationModify): Promise<boolean> {
+    public async onEnable(): Promise<boolean> {
         Global.app = this;
         return Promise.resolve(true);
     }
 
     public async executeLivechatBlockActionHandler(context: UIKitLivechatBlockInteractionContext,
                                                    read: IRead,
-                                                   http: IHttp,
-                                                   persistence: IPersistence,
+                                                   // tslint:disable-next-line: variable-name
+                                                   _http: IHttp,
```

Comment:
I think we don't need `// tslint:disable-next-line: variable-name` if we have already added `_` as a prefix to the variable name. 

Can you please remove it from all the places?